collect_app/src/main/java/org/odk/collect/android/application/initialization/migration/MigrationUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.application.initialization.migration;
```

Comment:
This and `ValueTranslator` feel somewhat disjoint from the migration tools in `org.odk.collect.android.preferences`. It would be good to consolidate them in your follow-up.

Reply:
@lognaturel they fit the structure that #3813 introduces so should fit in when these are merged 😀

Reply:
Oh great, I only went for a skim on that one since it was already reviewed!